pkg/apiserver/server.go

```diff
@@ -87,7 +88,13 @@ func Start(params *APIServerParams) {
 	}
 
 	if !util.IsHelmManaged() {
-		if err := operator.Start(params.AutocreateClusterToken); err != nil {
+		client := &client.Client{
```

Comment:
i don't see any values passed in this context, why can't this be initialized inside the operator?